src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.TargetFrameworkInference.targets

```diff
@@ -114,6 +106,19 @@ Copyright (c) .NET Foundation. All rights reserved.
                  FormatArguments="$([MSBuild]::Escape('$(TargetFramework)'))" />
   </Target>
 
+  <Target Name="_CheckForUnsupportedTargetPlatformIdentifier"
+          BeforeTargets="_CheckForInvalidConfigurationAndPlatform;RunResolvePackageDependencies;GetFrameworkPaths;GetReferenceAssemblyPaths"
+          Condition="'$(TargetPlatformIdentifier)' != ''" >
+
+    <PropertyGroup>
+      <TargetPlatformSupported Condition="'$(TargetPlatformIdentifier)' == 'Windows'">true</TargetPlatformSupported>
+    </PropertyGroup>
+
+    <NETSdkError Condition="'$(TargetPlatformSupported)' != 'true'"
+                 ResourceName="UnsupportedTargetPlatformIdentifier"
+                 FormatArguments="$([MSBuild]::Escape('$(TargetPlatformIdentifier)'))" />
```

Comment:
I may have previously left the same feedback, but is there a reason to escape this value?